### DISCUSSION TITLE
GoPro 12 Black dual audio channels

### DISCUSSION THREAD

**Discussion by gregb886** [gopro/labs#543] [General]
I have a 10 Black & Medial Mod and I have been reading reviews that say the 12 Black & Media Mod can record dual audio channels that can be edited later (not mixed into both channels in the camera).
Is that correct? (other reviews mention the need for the Max Lens Mod 2.0)

I would like to have external mic feed to the media mod on one channel and audio from the camera standard rear or front mic recorded on separate channels to I could balance the levels, ideally with standard or powered mic+ settings in the camera or by editing later.

Can the 12 Black with the Media Mod I bought for my 10 Black do this separate dual channel recording and does the 12 offer audio level settings for each mic?

**Comment by dnewman-gpsw** [comment]
Not in my experience, I don't know if that is possible. I did test that you can use the Line-In from MediaMod and BLE audio, resulting in two tracks.  I think the second track is only when using BLE audio input, it least in today's firmware.

**Comment by gregb886** [comment]
Thanks, David.
BLE audio would not be practical for me but I’ll watch for firmware updates & Labs firmware.
For now I will have to continue with using a mixer to set my two mono input levels before the line in to the media mod with my 10.

Cheers

Greg Burrowes
0404 182 428
________________________________
